doc/docs/filters.rst

```diff
@@ -42,6 +42,19 @@ If you want to write your own filter, have a look at :doc:`Write your own filter
 <filterdevelopment>`.
 
 
+Common options
+==============
+
+All filters support these options:
```

Comment:
This is wrong, it's not an option of the filter's constructor but an option of the function to find the filter. Ditto for the others.

Reply:
Good catch!

Reply:
Is there a section that would make more sense to put these in?  Or perhaps just a different wording to make it clear that the options aren't part of the filters themselves?